### ISSUE TITLE
superset db upgrade: No module named 'wtforms.compat'

### ISSUE THREAD

**Issue by MordicusEtCubitus** [apache/superset#17364]
Dears,

First thanks for this great software.
I've just installed Superset 1.3.2 in a fresh conda environment, and I've got `No module named 'wtforms.compat'` error when running `superset db upgrade` after installation.

```bash
conda create --name superset python=3.8  # this install 3.8.12
conda activate superset
pip install apache-superset
superset db upgrade
```
This produce:
```

Traceback (most recent call last):
  File "/home/user/anaconda3/envs/Superset/bin/superset", line 5, in <module>
    from superset.cli import superset
  File "/home/user/anaconda3/envs/Superset/lib/python3.8/site-packages/superset/__init__.py", line 21, in <module>
    from superset.app import create_app
  File "/home/user/anaconda3/envs/Superset/lib/python3.8/site-packages/superset/app.py", line 23, in <module>
    from superset.initialization import SupersetAppInitializer
  File "/home/user/anaconda3/envs/Superset/lib/python3.8/site-packages/superset/initialization/__init__.py", line 26, in <module>
    from flask_appbuilder import expose, IndexView
  File "/home/user/anaconda3/envs/Superset/lib/python3.8/site-packages/flask_appbuilder/__init__.py", line 6, in <module>
    from .base import AppBuilder  # noqa: F401
  File "/home/user/anaconda3/envs/Superset/lib/python3.8/site-packages/flask_appbuilder/base.py", line 8, in <module>
    from .api.manager import OpenApiManager
  File "/home/user/anaconda3/envs/Superset/lib/python3.8/site-packages/flask_appbuilder/api/manager.py", line 8, in <module>
    from flask_appbuilder.baseviews import BaseView
  File "/home/user/anaconda3/envs/Superset/lib/python3.8/site-packages/flask_appbuilder/baseviews.py", line 21, in <module>
    from .forms import GeneralModelConverter
  File "/home/user/anaconda3/envs/Superset/lib/python3.8/site-packages/flask_appbuilder/forms.py", line 16, in <module>
    from .fields import EnumField, QuerySelectField, QuerySelectMultipleField
  File "/home/user/anaconda3/envs/Superset/lib/python3.8/site-packages/flask_appbuilder/fields.py", line 6, in <module>
    from wtforms.compat import string_types, text_type
ModuleNotFoundError: No module named 'wtforms.compat
```

I've found a similar issue with wtforms_sqlachemy here, https://github.com/wtforms/wtforms-sqlalchemy/issues/29, but it is not involved in the import traceback

Could you help ?

With kind regards,

**Comment by MordicusEtCubitus** [comment]
I've just identified that wtforms.compat has been removed in wtforms 3.0
Maybe dependencies should be updated to require wtforms < 3.0 or the code updated...

**Comment by ian-r-johnson** [comment]
Looks like there's a fix coming from Flask App builder.

https://github.com/dpgaspar/Flask-AppBuilder/issues/1732

**Comment by shawnzhu** [comment]
I've tested the fix of #1732 in my superset (v1.3.2) with FAB v3.4.0 and :point_up: problem is resolved, plus a new OAuth signin/logout flow.
